app/controllers/reviews_controller.rb

```diff
@@ -10,6 +10,27 @@ def create
     redirect_to "/shelters/#{params[:shelter_id]}"
   end
 
+  def edit
+    @review = Review.find(params[:review_id])
+  end
+
+  def update
+    review = Review.find(params[:review_id])
+    if review.update(review_params) == false
+      flash[:notice] = "A required field is missing; please fill out all fields!"
```

Comment:
We might not need the flash html in the show view; but I couldn't figure out how to have the message show/render through here.  Switch it out if you can.

Reply:
We actually need to add the flash message syntax to the application.html, but I will get that added in.